benchmarks/sparsereg_run.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -xe
+
+device=cpu
+N=100
+benchmark_dir=$( cd $(dirname "$0") ; pwd -P )
```

Comment:
@fehiepsi - here's a driver script to run the benchmarks..could you run this with the same system config as the remaining ones? Let me fix the issue with the progress bar first.